java/src/jmri/jmrit/vsdecoder/Diesel3Sound.java

```diff
@@ -15,14 +15,12 @@
  * for more details.
  * <P>
  *
- * @author   Mark Underwood Copyright (C) 2011
+ * @author   Mark Underwood Copyright (C) 2011, 2018
```

Comment:
@klk32003 Unless you are also @msunderwd, please do not extend his copyright to cover your changes. Please include yourself as another author using a second `@author` tag.

If you are msunderwd as well, is there a technical concern that leads you to use two accounts?